Infra/Indexers/indexer_config.js

```diff
@@ -0,0 +1,58 @@
+const USDC_ADDRESS =
+  "0x53c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8";
+
+// hash.getSelectorFromName("Transfer")
+const TRANSFER_EVENT_KEY =
+  "0x99cd8bde557814842a3121e8ddfd433a539b8c9f14bf31ebf108d12e6196e9";
+
+export const usdcTransferFilter = {
+  header: {
+    weak: true,
+  },
+  events: [
+    {
+      fromAddress: USDC_ADDRESS,
+      keys: [TRANSFER_EVENT_KEY],
+      includeReceipt: false,
+    },
+  ],
+};
+
+export function decodeUSDCTransfer({ header, events }) {
+  const { blockNumber, blockHash } = header;
+  return events.map(({ event, transaction }) => {
```

Comment:
```suggestion
  return (events ?? []).map(({ event, transaction }) => {
```

`events` can be undefined if there's many blocks without any data.